### ISSUE TITLE
I have a problem with the audio metering.

### ISSUE THREAD

**Issue by ghost** [shadow-app/flutter_audio_recorder#17]
Hi bro, my app is using this lib, and I used the audio metering in the lib to describe sound effects when recording. But I found that when I spoke out loud, the peakPower or averagePower level did not change much compared to the silent moment. I want to ask if peakPower and averagePower are volume-related functions when recording or is it something else. Or do you have any good suggestions for describing the volume level when recording like Ok Google? Thanks for your reading.

**Comment by anup-geeky** [comment]
@huynhphu-pufu  
have you got any solution?
I want to show sound effects when audio recording. but for that i want list of peakPower.

**Comment by ghost** [comment]
@anup-geeky For me, I need to record and recognize spoken English word, so I used a plugin, which paid me the volume level when I spoke and realized the English word I just said. It is speech_to_text. As for you, I don't know what is your needs, but you can try the plugin I just mentioned or flutter_sound.

**Comment by ariefwijaya** [comment]
@huynhphu-pufu  but for my case , `flutter_sound` only support API 24+ :(

**Comment by ghost** [comment]
@ariefwijaya yeah, my PM also wants my apps support at least API 21, so I used speech_to_text for my purpose.

**Comment by tuanbs** [comment]
> @anup-geeky For me, I need to record and recognize spoken English word, so I used a plugin, which paid me the volume level when I spoke and realized the English word I just said. It is speech_to_text. As for you, I don't know what is your needs, but you can try the plugin I just mentioned or flutter_sound.

Can you please tell me how to do the recording audio and speech recognition at the same time? I used `audio_recorder` and `flutter_speech` packages but the iOS app sometimes crashed and on Android it doesn't support audio recording while doing speech recognition.
